src/basic/btrfs-util.c

```diff
@@ -954,9 +955,8 @@ int btrfs_resize_loopback_fd(int fd, uint64_t new_size, bool grow_only) {
         if (grow_only && new_size < (uint64_t) st.st_size)
                 return -EINVAL;
 
-        if (asprintf(&loop, "/dev/block/%u:%u", major(dev), minor(dev)) < 0)
-                return -ENOMEM;
-        loop_fd = open(loop, O_RDWR|O_CLOEXEC|O_NOCTTY);
+        xsprintf_sys_block_path(p, "", dev);
```

Comment:
hmm, I think it would be prettier if this macro would also accept NULL here as a suffix... Yes, this would mean we'd have to use %s to format the suffix, but that's not too bad, if we call sprintf anyway...

Reply:
OK.